PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
+## Smart Shopping List Resources
 
 - Site: {PRODUCTION_URL}
 - Repo: {REPO_URL}
 - Clone URL: {REPO_CLONE_URL}
 - Issue list: {PROJECT_BOARD_URL}
 - Database: {CONSOLE_URL}
 
-### Project cadence & duration
+## Program Agenda & Curriculum
 
-The project is expected to take 40 hours per participant, spread over 8 weeks.
-
-Each week, the team of 4 developers will split into 2 pairs of 2 developers each. Each pair will take on a story or task from the issues list to work on together that week.
-
-Pairings will go as follows:
-
-#### Week 1, {DATES_WEEK_1}
-
-1. {DEV_1} & {DEV_2}
-2. {DEV_3} & {DEV_4}
-
-#### Week 2, {DATES_WEEK_2}
-
-1. {DEV_2} & {DEV_3}
-2. {DEV_1} & {DEV_4}
-
-#### Week 3, {DATES_WEEK_3}
-
-1. {DEV_3} & {DEV_1}
-2. {DEV_4} & {DEV_2}
-
-#### Week 4, {DATES_WEEK_4}
-
-1. {DEV_1} & {DEV_2}
-2. {DEV_3} & {DEV_4}
-
-#### Week 5, {DATES_WEEK_5}
-
-1. {DEV_2} & {DEV_3}
-2. {DEV_1} & {DEV_4}
-
-#### Week 6, {DATES_WEEK_6}
-
-1. {DEV_3} & {DEV_1}
-2. {DEV_4} & {DEV_2}
-
-#### Week 7, {DATES_WEEK_7}
-
-1. {DEV_1} & {DEV_2}
-2. {DEV_3} & {DEV_4}
-
-#### Week 8, {DATES_WEEK_8}
-
-1. {DEV_2} & {DEV_3}
-2. {DEV_1} & {DEV_4}
+### Week 1, {DATES_WEEK_1}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                                    |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| {DEV_1} & {DEV_2}     | [2. As a user, I want to read all of my shopping lists and the items in them]({REPO_URL}/issues/2)                                          |
+| {DEV_3} & {DEV_4}     | [3. As a user, I want to be able to navigate to all the pages in the application by clicking the links in the nav bar]({REPO_URL}/issues/3) |
+
+| 📖 Learning modules                                                                                                                                                                 | Description |
+| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| [Git Presentation](https://youtu.be/zeC5iwsHllk) ([slides](https://docs.google.com/presentation/d/1onRsF-5NQbky8BOac5cGsJunjBisvauQPEC4hOWPaGI/edit#slide=id.p))                    |
+| [Code Review Presentation](https://youtu.be/QcItLjFJ_Us) ([slides](https://docs.google.com/presentation/d/1GP_X91rd3W50GN6FjcLfAge8SD0O45qe7qdtHPO_D2I/edit?pli=1#slide=id.p))      |
+| [Communication for Devs Presentation](https://youtu.be/LVjG3qxtQX0) ([slides](https://docs.google.com/presentation/d/14kJTPHCVHsEmzqOLTqKCgL3QrwR7_BnIpN9oGJraT4c/edit#slide=id.p)) |
+| [Pair Programming Presentation](https://youtu.be/yFVJ2gph_Uc) (slides)                                                                                                              |
```

Comment:
**Possible Description:** In this module, you'll learn what pair programming is and what is expected of you while pairing. 

_Bulleted list, if needed:_

You'll learn:

- the various roles
- how to pair remotely
- how to have the proper mindset and attitude 
- useful resources